packages/core/src/routes/applications/application.ts

```diff
@@ -314,4 +314,21 @@ export default function applicationRoutes<T extends AuthedRouter>(
       return next();
     }
   );
+
+  if (EnvSet.values.isDevFeaturesEnabled) {
+    router.get(
+      '/applications/configs/protected-app-default-domain',
+      koaGuard({
+        response: object({ domain: string() }),
+        status: [200],
+      }),
```

Comment:
should we put this under a global route like `/configs`?

Reply:
updated as `/application-configs`